db/migrate/20190422200243_add_last_reminded_at_index.rb

```diff
@@ -0,0 +1,13 @@
+class AddLastRemindedAtIndex < ActiveRecord::Migration[5.2]
+  def up
+    execute <<~SQL
+    CREATE UNIQUE INDEX idx_last_reminded_at
```

Comment:
Usually we include the table name in the index name so that we can quickly tell which table it belongs to.